docs/manual/docs/cmd/teams/deeplink/tab-generate.md

```diff
@@ -0,0 +1,40 @@
+# teams deeplink tab generate
+
+Generates a Microsoft Teams deep link from an existing Tab in a Channel
+
+## Usage
+
+```sh
+teams deeplink tab generate [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`| output usage information
+`-i, --teamId <teamId>`|The ID of the team where the tab exists
+`-c, --channelId <channelId>`|The ID of the channel where the tab exists
+`-t, --tabId <tabId>`|The ID of the tab to generate the deep link from
+`-l, --label <label>`|The label to use in the deep link
+`-m, --tabType <TabTypeOptions>`|The tab type. Allowed values `Static`, `Configurable`
```

Comment:
Lets change `<TabTypeOptions>` to `<tabTypeOptions>`, note the lowercase leading character